### PR TITLE
Fixes related to MPL drawer

### PR DESCRIPTION
**Pull request by MatteoRobbiati**
Thanks @sergiomtzlosa for the fixes already done in #1438. We can keep that PR to solve the coverage problems and use this one to fix asap the errors.

Checklist:
- [x] Reviewers confirm new code works as expected.
- [x] Tests are passing.
- [x] Coverage does not decrease (this is expected to decrease since we will fix it in #1438).
- [x] Documentation is updated.

### CODE REVIEW

**Comment by sergiomtzlosa** [comment]
great @MatteoRobbiati just let me know to get hands on

File: src/qibo/ui/mpldrawer.py [hunk 1]
```diff
@@ -610,13 +593,13 @@ def _process_gates(array_gates):
...
             for qbit in gate._target_qubits:
-                if qbit is tuple:
+                if type(qbit) is tuple:
                     item += ("q_" + str(qbit[0]),)
                 else:
                     item += ("q_" + str(qbit),)
 
             for qbit in gate._control_qubits:
-                if qbit is tuple:
+                if type(qbit) is tuple:
```
Comment (renatomello) [inline src/qibo/ui/mpldrawer.py:602]:
```suggestion
                if isinstance(qbit, tuple):
```

File: src/qibo/ui/mpldrawer.py [hunk 2]
```diff
@@ -610,13 +593,13 @@ def _process_gates(array_gates):
...
                     item += ("q_" + str(qbit[0]),)
                 else:
                     item += ("q_" + str(qbit),)
```
Comment (renatomello) [inline src/qibo/ui/mpldrawer.py:605]:
```suggestion
                item_add = ("q_" + str(qbit[0]),) if isinstance(qbit, tuple) else  ("q_" + str(qbit),)
                item += item_add
```

Comment (sergiomtzlosa) [reply src/qibo/ui/mpldrawer.py:605]: thanks @renatomello , As this is an enhance and not a fix I add this in https://github.com/qiboteam/qibo/pull/1438

Comment (MatteoRobbiati) [reply src/qibo/ui/mpldrawer.py:605]: As you wish @sergiomtzlosa, we can also just accept this suggestion here and keep the other one to fix the coverage :) Let me know.

Comment (sergiomtzlosa) [reply src/qibo/ui/mpldrawer.py:605]: ok @MatteoRobbiati just accept the fix, it is fine for me 👍

**Review by sergiomtzlosa — APPROVED** [review]

**Review by renatomello — APPROVED** [review]

**Comment by renatomello** [comment]
@MatteoRobbiati I can't merge it since coverage is not passing. Idk if you have the right level of authorization for that or if it's just @scarrazza

**Comment by scarrazza** [comment]
Coverage is failing, correct?

**Comment by renatomello** [comment]
> Coverage is failing, correct?

If I understood correctly, coverage fix is being implemented in #1438

**Comment by sergiomtzlosa** [comment]
> > Coverage is failing, correct?
> 
> If I understood correctly, coverage fix is being implemented in #1438

Yes, you are right

**Comment by MatteoRobbiati** [comment]
Yes, correct. With this we fix errors and with the other one we can fix coverage. If you prefer, we can fix everything there, but that will take a bit more time.

**Comment by renatomello** [comment]
> Yes, correct. With this we fix errors and with the other one we can fix coverage. If you prefer, we can fix everything there, but that will take a bit more time.

I'd prefer if this is merged as is, so other PRs can move forward while #1438 fixes the coverage.

**Comment by scarrazza** [comment]
I agree, thanks.

**Pull request merged**